### PR TITLE
Refactor/uncrustify

### PR DESCRIPTION
**Pull request by dundargoc**

### CODE REVIEW

File: src/nvim/math.c [hunk 1]
```diff
@@ -2,6 +2,7 @@
 // it. PVS-Studio Static Code Analyzer for C, C++ and C#: http://www.viva64.com
 
 #include <math.h>
+
```
Comment (bfredl) [inline src/nvim/math.c:5]: This looks random. not a big deal tho.

Comment (dundargoc) [reply src/nvim/math.c:5]: "There are no accidents" - Master Oogway

Not random, I've enabled a flag that separates the <>-includes from ""-includes. Can turn it off of course.

Comment (dundargoc) [reply src/nvim/math.c:5]: It's a bug. The filename is math.c and it thinks that it's its own header.

**Review by justinmk — APPROVED** [review]
This is a big improvement, thank you.

**Pull request merged**